cmd/dinghy.go

```diff
@@ -57,11 +58,17 @@ func newRedisOptions(redisOptions settings.Redis) *redis.Options {
 
 func Setup() (*logr.Logger, *web.WebAPI) {
 	log := logr.New()
-	config, err := settings.LoadSettings()
+	sourceConfiguration, err := settings.LoadSettings()
 	if err != nil {
 		log.Fatalf("failed to load configuration: %s", err.Error())
 	}
 
+	// We need to initialize the configuration for the start-up.
+	config, err := sourceConfiguration.LoadSetupSettings()
+	if err != nil {
+		log.Fatalf("%s", err.Error())
```

Comment:
Is there more context we can provide here? Also I think we _should_ be able to do `log.Fatalf("%w", err)` to wrap the error.

Reply:
Done ✅ !!